### PR TITLE
feat: command panel for tabs

### PR DESCRIPTION
**Pull request by arashsheyda**
related to #73 

I wasn't sure about this😁. Is it going to be only for tab switching, or is it going to have other things too?should it be changed to group-items?(tabs group, docs group ....)

### CODE REVIEW

**Comment by arashsheyda** [comment]
I wasn't sure about this😁. Is it going to be only for tab switching, or is it going to have other things too?should it be changed to group-items?(tabs group, docs group ....)

File: packages/devtools-ui-kit/src/components/NDialog.vue [hunk 1]
```diff
@@ -21,7 +21,7 @@ const emit = defineEmits<{
 const show = useVModel(props, 'modelValue', emit, { passive: true })
 const card = ref(null)
 
-const { activate, deactivate } = useFocusTrap(card, { immediate: false })
+const { activate, deactivate } = useFocusTrap(card.value, { immediate: false })
```
Comment (antfu) [inline packages/devtools-ui-kit/src/components/NDialog.vue:24]: This does not seem right, we are passing the ref on purpose as it will change reactively.

Comment (arashsheyda) [reply packages/devtools-ui-kit/src/components/NDialog.vue:24]: @antfu when it was `useFocusTrap(card, { immediate: false })` i was getting this error:
`Uncaught (in promise) Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times`

Comment (arashsheyda) [reply packages/devtools-ui-kit/src/components/NDialog.vue:24]: can we use `<Teleport v-lazy-show="show" to="body">` ?

Comment (arashsheyda) [reply packages/devtools-ui-kit/src/components/NDialog.vue]: @antfu changed it to v-lazy-show, sorry in advance if i messit up,
but still there is a problem with the focusing on input for first time that opens


https://github.com/nuxt/devtools/assets/38922203/c40af9ea-c489-48d3-a1b1-c85dff12b2a3

**Pull request merged**